src/__tests__/organization.tests.ts

```diff
@@ -0,0 +1,319 @@
+        });
+    
+        it("When updating new oraganization with empty address, Expect to return HTTP Status 400", async () => {
+          //Arrange
+          let oraganizationApi = new OrganizationApi(configurationWithAuthHeader);
+          //Act
+          let payload = TdeiObjectFaker.getOrganization();
+          payload.address = '';
+    
+          const oraganizationResponse = oraganizationApi.updateOrganization(payload);
+    
+          //Assert
+          await expect(oraganizationResponse).rejects.toMatchObject({response: { status: 400 } });
+        });
+    
+        it('When updating new organization with invalid polygon, Expect to return HTTP Status 400', async () => {
+          //Arrange
+          let oraganizationApi = new OrganizationApi(configurationWithAuthHeader);
+          //Act
+          let payload = TdeiObjectFaker.getOrganization();
+          payload.polygon = TdeiObjectFaker.getInvalidPolygon();
+    
+          const oraganizationResponse = oraganizationApi.updateOrganization(payload);
+    
+          //Assert
+          await expect(oraganizationResponse).rejects.toMatchObject({ response: { status: 400 } });
+         });
+        });
+      });
+
+      describe('Get Station', () => {
+        describe('Auth', () => {
+          it("When no auth token provided, Expect to return HTTP status 401", async () => {
+            //Arrange
+            let oraganizationApi = new OrganizationApi(configurationWithoutAuthHeader);
+            //Act
+            const request = async () => {
+              await oraganizationApi.getOrganization();
+            }
+            //Assert
+            expect(request()).rejects.toMatchObject({response: { status: 401 } })
+          });
+        });
+
+        describe('Functional', () => {
+          it('When searched without filters, Expect to return list of organizations of type OrganizationList', async () => {
+            //Arrange
+            let oraganizationApi = new OrganizationApi(configurationWithAuthHeader);
+
+            //Act
+            const oraganizationResponse = await oraganizationApi.getOrganization();
+
+            //Assert
+            expect(oraganizationResponse.status).toBe(200);
+            expect(oraganizationResponse.data).toBeInstanceOf(Array);
+            });
+
+            it('When searched with tdei_org_id filter, Expect to return list of organizations matching filter', async () => {
+            //Arrange
+            let oraganizationApi = new OrganizationApi(configurationWithAuthHeader);
+
+            //Act
+            const oraganizationResponse = await oraganizationApi.getOrganization(seederData?.organizationId, undefined, undefined, undefined, undefined);
+
+            const data = oraganizationResponse.data;
+
+            //Assert
+              expect(oraganizationResponse.status).toBe(200);
+              expect(Array.isArray(data)).toBe(true);
+              expect(data).toBeInstanceOf(Array);
```

Comment:
Duplicate assertion

Reply:
assertion is similar to the one in pathways.

Reply:
that is double assertion. Need to be removed in pathways as well.